packages/myst-parser/src/transforms/listItemParagraphs.ts

```diff
@@ -0,0 +1,65 @@
+import type { GenericNode, GenericParent } from 'myst-common';
+import { selectAll } from 'unist-util-select';
+
+export function listItemParagraphsTransform(tree: GenericParent) {
```

Comment:
This is the transform that is added at the end of a markdown parse. It is optional, but turned on by default to be compatible with mdast.